tests/Config/RedisConnectionConfigTest.php

```
@@ -125,28 +124,41 @@
-        self::assertSame(0, $config->getRetryInterval());
+    /**
+     * @dataProvider zeroValueProvider
+     */
+    public function testConstructorAcceptsZeroValues(string $parameter, mixed $expected): void
+    {
+        $config = match ($parameter) {
+            'timeout' => new RedisConnectionConfig('localhost', 6379, 0.0),
+            'readTimeout' => new RedisConnectionConfig('localhost', 6379, 2.5, null, 0, 'session:', false, 100, 0.0),
+            'maxRetries' => new RedisConnectionConfig('localhost', 6379, 2.5, null, 0, 'session:', false, 100, 2.5, 0),
+            'retryInterval' => new RedisConnectionConfig('localhost', 6379, 2.5, null, 0, 'session:', false, 0),
+            default => throw new \InvalidArgumentException("Unknown parameter: {$parameter}"),
```

PHPMD (2.15.0)

Missing class import via use statement (line '137', column '34'). (undefined)

(MissingImport)